idea-plugin/build.gradle.kts

```diff
@@ -47,9 +46,13 @@ dependencies {
 
     // https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin.html#setting-up-intellij-platform
     intellijPlatform {
-        intellijIdeaCommunity("2024.1")
+        intellijIdeaCommunity(libs.versions.idea)
         instrumentationTools()
         zipSigner()
+
+        // dependency plugin id for https://plugins.jetbrains.com/plugin/6954-kotlin
+        bundledPlugin(libs.kotlin.stdlib.map(Dependency::getGroup))
+        localPlugin(projects.components.psi)
```

Comment:
@Goooler maybe this break release

https://github.com/ComposeGears/Valkyrie/actions/runs/10330536890

Package 'io.github.composegears.valkyrie.psi' is not found
          Package 'io.github.composegears.valkyrie.psi' is not found along with its 2 classes.
          Probably the package 'io.github.composegears.valkyrie.psi' belongs to a library or dependency that is not resolved by the checker.
          It is also possible, however, that this package was actually removed from a dependency causing the detected problems. Access to unresolved classes at runtime may lead to **NoSuchClassError**.
          The following classes of 'io.github.composegears.valkyrie.psi' are not resolved:
            Class io.github.composegears.valkyrie.psi.IconPackInfo is referenced in
              io.github.composegears.valkyrie.ui.screen.mode.iconpack.existingpack.ui.viewmodel.ExistingPackViewModel.onChooseFile(Path path, Project project) : void

Reply:
56e2984353d55530f0cb3cac5410a3a79b9944ae

Wait https://github.com/ComposeGears/Valkyrie/actions/runs/10330644000/job/28599926138.